dev/generator/main.go

```diff
@@ -113,23 +113,26 @@ func buildPackage(packagesBasePath string, p util.Package) error {
 	if err != nil {
 		return err
 	}
-	err = os.Chdir(packagesBasePath)
-	if err != nil {
-		return err
-	}
-	defer os.Chdir(currentPath)
 
 	// Checks if the package is valid
 	err = p.Validate()
 	if err != nil {
 		return fmt.Errorf("Invalid package: %s: %s", p.GetPath(), err)
 	}
 
+	p.BasePath = currentPath + "/" + packagesBasePath + "/" + p.GetPath()
+
 	err = p.LoadAssets(p.GetPath())
 	if err != nil {
 		return err
 	}
 
+	err = os.Chdir(packagesBasePath)
```

Comment:
Not blocking, but I'm not convinced if the app should change directory.

Reply:
This is exactly what I'm trying to get rid off. Following up with more PRs.